docs/sources/enterprise/saml.md

```diff
@@ -204,6 +204,61 @@ org_mapping = Engineering:2, Sales:3
 allowed_organizations = Engineering, Sales
 ```
 
+## Set up SAML with Okta
+
+To configure SAML integration with Okta, create integration inside the Okta org first.
+
+1. Log in to the [Okta portal](https://login.okta.com/).
+1. Go to the Admin Console in your Okta org by clicking **Admin** in the upper-right corner. If you are in the Developer Console, click **Developer Console** in the upper-left corner and then click **Classic UI** to switch over to the Admin Console.
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Click **Add Application**.
+1. Click **Create New App** to start the Application Integration Wizard.
+1. Choose **Web** as a platform.
+1. Select **SAML 2.0** in the Sign on method section.
+1. Click **Create**.
+1. On the **General Settings** tab, enter a name for your Grafana integration and optionally upload a logo.
+1. On the **Configure SAML** tab, enter the SAML information related to your Grafana instance:
+    - In the **Single sign on URL** field, use the `/saml/acs` endpoint URL of your Grafana instance, for example, `https://grafana.example.com/saml/acs`.
+    - In the **Audience URI (SP Entity ID)** field, use the `/saml/metadata` endpoint URL, for example, `https://grafana.example.com/saml/metadata`.
+    - Leave the default values for **Name ID format** and **Application username**.
+    - In the **ATTRIBUTE STATEMENTS (OPTIONAL)** section, enter the SAML attributes to be shared with Grafana, for example:
+      
+      | Attribute name (in Grafana) | Value (in Okta profile)                |
+      | --------------------------- | -------------------------------------- |
+      | Login                       | `user.login`                           |
+      | Email                       | `user.email`                           |
+      | DisplayName                 | `user.firstName + " " + user.lastName` |
+    
+    - In the **GROUP ATTRIBUTE STATEMENTS (OPTIONAL)** section, enter a group attribute name (for example, `Group`) and set filter to `Matches regex .*` to return all user groups.
+1. Click **Next**.
+1. On the final Feedback tab, fill the form and click **Finish**.
+
+Once application is created, configure Grafana to use it for SAML authentication.
+
+1. In the `[auth.saml]` section in the grafana configuration file, set [`enabled`]({{< relref "./enterprise-configuration.md#enabled" >}}) to `true`.
```

Comment:
```suggestion
1. In the `[auth.saml]` section in the Grafana configuration file, set [`enabled`]({{< relref "./enterprise-configuration.md#enabled" >}}) to `true`.
```

Reply:
"enabled" is not descriptive link text. I recommend linking to the auth.saml section, or add a sentence to the end saying, "This section is described in the [auth.saml](link) section of [Configuration](Link to configuration.md).